packages/klighd-core/src/views.tsx

```diff
@@ -187,10 +214,14 @@ export class KNodeView implements IView {
             result.push(rendering)
         } else {
             ctx.positions.pop()
-            return <g>
-                {ctx.titles.pop() ?? []}
-                {ctx.renderChildren(node)}
-            </g>
+            const titles = ctx.exitTitleScope()
+            const childRenderings = ctx.renderChildren(node)
+            ctx.popEffectiveZoom()
+            const ret =  <g><g transform={transformation}>
+                {titles}
+                {childRenderings}
+            </g></g>
+            return ret
```

Comment:
or just return <g>...

Also, why is there two <g> immedeatly nested with the outer one seemingly doing nothing now?

Reply:
The separate return is a left over from println debugging.
The second group is necessary as sprotty overrides the transform attribute to apply the elements offset, which would otherwise override the transformation I set here